module-2/backend/week_one.md

```diff
@@ -1,32 +1,146 @@
-## Week One - Module 2 Recap
-
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!). 
-
-Note: When you're done, submit a PR. 
-
-1. List the five common HTTP verbs and what the purpose is of each verb.
-2. What is Sinatra?
-4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
-6. What types of variables are accessible in our view templates without explicitly passing them?
-7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
-  
-  ```ruby
-  get '/horses' do
-    erb :index
+Week One - Module 2 Recap
+
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!).
+
+Note: When you're done, submit a PR.
+
+List the five common HTTP verbs and what the purpose is of each verb.
+1. Get - retrieve data
+* POST - sending data to the server
+* DELETE - remove data
+* PUT - will replace/update
+* cant think of another :(

```

Comment:
PATCH is an alternative for PUT. Generally they are interchangeable, but sticklers will tell you that PUT replaces a full record and PATCH updates part of a record.